tests/performance-tests/src/tests/public/findStatisticsPage.test.ts

```diff
@@ -55,15 +55,12 @@ const performTest = () => {
     getReleaseFailureCount.add(1);
     errorRate.add(1);
     fail(`Failure to get Find Statistics page - ${JSON.stringify(e)}`);
-    return;
```

Comment:
I think that we actually want to return early here and not proceed onto the following `check(response` statement, as that will flag up additional failures on top of the ones we've just registered.

If it's linting that's advising us away from having this return statement, can we put the rest of the code in an "else" instead?

Reply:
There's no need to add a return statement here as it's unreachable. We're in the `catch` and `fail` will immediately throw an error, meaning anything below it will not get executed